autosarfactory/autosar_ui.py

```diff
@@ -37,13 +62,31 @@ def __initialize_ui(self):
 
         # set theme
         style = ThemedStyle(self.__root)
+        #print(style.themes)
+#        ['yaru', 'default', 'vista', 'classic', 'scidgreen', 'equilux', 'scidgrey', 'adapta', 'scidpink', 'scidmint', 'plastik', 'alt', 'clearlooks', 'itft1', 'smog', 'clam', 'scidsand', 'kroc', 'radiance', 'black', 'blue', 'arc', 'winxpblue', 'scidblue', 'ubuntu', 'keramik', 'winnative', 'elegance', 'aquativo', 'scidpurple', 'xpnative', 'breeze']
+        #style.theme_use("equilux") #- very slow for some reason
+
         style.theme_use('scidgreen')

```

Comment:
We can use self.__current_theme  directly here.

Reply:
No. That will not work. I tried that before already.